src/Job/GenerateStaticCacheJob.php

```diff
@@ -28,7 +28,6 @@ public function process()
             }
             $meta = Publisher::singleton()->publishURL($url, true);
             if (!empty($meta['success'])) {
-                $this->jobData->ProcessedURLs[$url] = $url;
```

Comment:
Why don't we need this anymore?

Reply:
https://github.com/silverstripe/silverstripe-staticpublishqueue/pull/127/files#r542996009

Reply:
Reverted